Applications/Spire/Source/SpireTester/ExportTableTester.cpp

```diff
@@ -1,40 +1,68 @@
 #include <doctest/doctest.h>
+#include "boost/date_time/c_local_time_adjustor.hpp"
+#include "boost/date_time/posix_time/posix_time.hpp"
+#include <locale>
 #include "Nexus/Definitions/SecuritySet.hpp"
 #include "Spire/Spire/ArrayTableModel.hpp"
 #include "Spire/Spire/ExportTable.hpp"
 #include "Spire/Ui/CustomQtVariants.hpp"
 
+using namespace boost;
+using namespace boost::gregorian;
 using namespace boost::posix_time;
 using namespace Nexus;
 using namespace Spire;
 
+auto to_string(ptime t) {
```

Comment:
What is the purpose of this function? Look at removing it.

Reply:
The purpose of this function is to display the date of the time. The function of `displayText(ptime time)` in `CustomQtVariants` doesn't show the date of the current day.
```
QString Spire::displayText(ptime time) {
  auto local_time = ToLocalTime(time);
  auto current_time = ToLocalTime(posix_time::second_clock::universal_time());
  if(local_time.date() == current_time.date()) {
    return QString::fromStdString(to_simple_string(local_time).substr(12));
  }
  return QString::fromStdString(to_simple_string(local_time));
}
```

Reply:
I will think about how to remove it.